README.md

```diff
@@ -0,0 +1,42 @@
+# ftrack OpenAssetIO Plugin
+
+A skeleton OpenAssetIO ManagerPlugin that binds ftrack to the API
+
+> **Warning**
+> This plugin is highly experimental, and subject to significant
+> change.
+
+## Getting started
+
+To use the plugin in an OpenAssetIO host, set (or append) the
+`OPENASSETIO_PLUGIN_PATH` env var to include the `python` directory in
+this checkout.
+
+The plugin requires `openassetio` to be available to python at runtime
+(see the [project documentation](https://github.com/OpenAssetIO/OpenAssetIO#getting-started)
+for more information).
+
+The plugin provides a manager with the identifier `com.ftrack`.
+
+## Testing
+
+The test fixtures take care of configuring the OpenAssetIO plugin search
+paths for you. Assuming your working directory is set to the root of
+this checkout:
+
+```bash
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+python -m pip install -r requirements.test.txt
```

Comment:
[consider][minor] Just to throw a couple of other options into the mix.

* The two requirements files can be placed on a single line, i.e. `pip install -r requirements.test.txt -r requirements.txt`; or
* The test requirements can refer to the main requirements, i.e. adding `-r ./requirements.txt` to the top of `requirements.test.txt`. This apparently works even if `pip install` is run from a different directory too (which kinda surprised me).  Then the command-line only needs the test requirements file.  Possibly obscures things slightly, though.

But very minor, happy as it is.

Reply:
@feltech Have a look at the structure now. I re-jigged it a bit to better match some of our existing projects.

Note in particular, I removed `openassetio` as a pip managed dependency until we have wheels actually published to PyPi, and tweaked the note in the README.